### PR TITLE
[Merged by Bors] - Add setup documentation for Alpine Linux

### PR DESCRIPTION
**Pull request by rdbo**
Adds documentation for setting up bevy on Alpine Linux and its derivatives.
It contains instructions on installing the required packages and also fixing runtime errors.

### CODE REVIEW

File: docs/linux_dependencies.md [hunk 1]
```diff
@@ -162,3 +162,19 @@ When using a NVIDIA GPU with the proprietary driver (eg. `x11-drivers/nvidia-dri
 sudo swupd bundle-add devpkg-alsa-lib
 sudo swupd bundle-add devpkg-libgudev
 ```
+
+## [Alpine Linux](https://alpinelinux.org/)
+
+Run the following command to install `GNU C compiler, standard C development libraries, pkg-config, X11 development libraries, ALSA development libraries, eudev development libraries`:
```
Comment (james7132) [inline docs/linux_dependencies.md:168]:
```suggestion
Run the following command to install `GNU C compiler, standard C development libraries, pkg-config, X11 development libraries, ALSA development libraries, eudev development libraries`:

```

File: docs/linux_dependencies.md [hunk 2]
```diff
@@ -162,3 +162,19 @@ When using a NVIDIA GPU with the proprietary driver (eg. `x11-drivers/nvidia-dri
...
+```sh
+sudo apk add gcc libc-dev pkgconf libx11-dev alsa-lib-dev eudev-dev
+```
+Install a GPU renderer for you graphics card. For Intel integrated GPUs:
```
Comment (james7132) [inline docs/linux_dependencies.md:172]:
```suggestion

Install a GPU renderer for you graphics card. For Intel integrated GPUs:

```

**Review by james7132 — CHANGES_REQUESTED** [review]
Please adhere to the style of the rest of the docs and add a blank line between code blocks and normal text.

**Comment by rdbo** [comment]
Added the blank lines, as requested.

**Review by james7132 — APPROVED** [review]

**Review by alice-i-cecile — APPROVED** [review]
Thanks :)

bors r+

**Pull request closed without merging**